### PR TITLE
[FSPE-6534] Changed input disabled styles

### PR DESCRIPTION
**Pull request by mattnickles**
https://ctl.atlassian.net/browse/FSPE-6534

* Added: Number input documentation now includes examples for Min, Max, Step, Required, Help, Message, and Error.
* Added: Number input now supports required attribute.
* Changed: File input disabled state is now consistent with button disabled state.
* Changed: Number input expanded disabled state is now consistent with button disabled state.
* Changed: Number input no longer renders semantic border and focus colors when disabled.
* Changed: Text input and Textarea disabled text color has been updated to reflect latest guidelines.
* Changed: Number input documentation text for disabled state has been improved.
* Fixed: Label wrapper label margin-bottom specificity issue.
* Fixed: Base Number Input did not support disabled button styles for min/max scenarios.

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1585/ for more details.

**Comment by janibolkvadze** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://20ca98d027bd9a6a85ca1bca2eec475032ca0398-chi-pr.enterprise.ctl.io" target="_blank"> https://20ca98d027bd9a6a85ca1bca2eec475032ca0398-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1588/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://625e4c8f36f6655c1e3587ec39716f92f14e2e5d-chi-pr.enterprise.ctl.io" target="_blank"> https://625e4c8f36f6655c1e3587ec39716f92f14e2e5d-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1593/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://8de984e57cf989cdc309cf3b9043f1843bc74f65-chi-pr.enterprise.ctl.io" target="_blank"> https://8de984e57cf989cdc309cf3b9043f1843bc74f65-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1595/ for more details.

**Review by jllr — APPROVED** [review]

File: src/website/views/components/forms/number-input/_examples.pug [hunk 1]
```diff
@@ -71,6 +75,248 @@ h3 Disabled
...
+            <chi-icon icon="circle-info-outline"></chi-icon>
+          </chi-button>
+          <chi-popover id="example__help-popover" position="top" variant="text" arrow reference="#example__help-button">
+            Helpful information goes here.
+          </chi-popover>
+        </div>
+      </div>
+      <chi-number-input id="unique-id-he1"></chi-number-input>
+
+      <script>chi.popover(document.getElementById('example__help-button'));</script>
```
Comment (janibolkvadze) [inline src/website/views/components/forms/number-input/_examples.pug:204]:
```suggestion
      <script>
        document
          .querySelector("#example__help-button")
          .addEventListener("click", function () {
            var popoverElem = document.querySelector("#example__help-popover");

            popoverElem.toggle();
          });
      </script>
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://a9f54559cec1589c944d94229166350ff7b1faf9-chi-pr.enterprise.ctl.io" target="_blank"> https://a9f54559cec1589c944d94229166350ff7b1faf9-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1596/ for more details.

**Review by janibolkvadze — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-6w6cds, https://ops.elasticbox.com/services/instances/i-imi73r, https://ops.elasticbox.com/services/instances/i-ppkuyz, https://ops.elasticbox.com/services/instances/i-vk5yh7

**Pull request merged**